peekingduck/pipeline/nodes/draw/blur_bbox.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Blur area bounded by bounding boxes over detected object
+"""
+
+from typing import Any, Dict, List
+import numpy as np
+from scipy.ndimage import gaussian_filter
+from peekingduck.pipeline.nodes.node import AbstractNode
+
+
+class Node(AbstractNode):  # pylint: disable=too-few-public-methods, duplicate-code
```

Comment:
Does the "duplicate-code" suppress the warning? If no, remove

Reply:
the comment is removed, as it does not work